netkan/netkan/cli.py

```diff
@@ -41,7 +50,8 @@ def netkan():
     help='Reduce message visibility timeout for testing',
 )
 @click.option('--key', envvar='SSH_KEY', required=True)
-def indexer(queue, metadata, token, repo, user, key, debug, timeout):
+def indexer(queue, metadata, token, repo, user, key,
+            status_db, debug, timeout):
```

Comment:
All of this file's functions now have a `status_db` parameter, but none of them seem to use it. Am I missing something, is there some way they get used implicitly?

Reply:
Yeah it could be dropped as passing through as an option won't work. The Status class is what needs it, but the click context isn't available until after. I made a comment regarding the table name. Probably worth dropping entirely as it really is just a dev thing.

Reply:
This one (the `status_db` param of `indexer`) still seems to be there, while the others are removed.

Reply:
Ahh late night coding. Glad I have a second set of eyes looking at this code!